lib/backends/sdc/volumes.js

```diff
@@ -153,15 +159,78 @@ function listVolumes(params, options, callback) {
     assert.object(options, 'options');
     assert.func(callback, 'callback');
 
+    var filterName;
+    var filters;
+    var filterValue;
     var log = options.log;
+    var reqParams = {};
+    var valIdx;
+
+    /*
+     * By default, we only list volumes that are ready. As far as I know, docker
+     * doesn't have a concept of a volume's state, and so doesn't provide any
+     * ability to filter on such a property.
```

Comment:
That's a great question. I think it's ok to leave it as is in this PR since it's not a change from the previous behavior, but it seems it would be a good candidate for a separate ticket and investigation. Thoughts?